students/yixingxu/lesson08/test_circle.py

```diff
@@ -0,0 +1,53 @@
+from circle import Circle
+
+def test_init():
+    c = Circle(4)
+    assert (c.radius == 4) is True
```

Comment:
you don't need `is True`, it is redundant; it's basically saying <something> is True is True

`assert c.radius == 4` is sufficient